studio.eufonia.EufoniaClient.yml

```diff
@@ -0,0 +1,79 @@
+id: studio.eufonia.EufoniaClient
+
+runtime: org.gnome.Platform
+runtime-version: '47'
+sdk: org.gnome.Sdk
+tags:
+  - proprietary
+
+command: eufonia-client
+finish-args:
+  - --device=dri # Hardware acceleration
+  - --device=input # Game controllers
```

Comment:
this won't work. It's not available anywhere.

Reply:
yes for now

Reply:
Okay! Thank you for the information